python/ray/train/_internal/data_config.py

```diff
@@ -46,6 +47,18 @@ def __init__(
             execution_options or DataConfig.default_ingest_options()
         )
 
+        self._num_train_cpus = 0.0
```

Comment:
Can we call this something like "num_reserved_cpus"? Because it seems useful even if Train/Tune is not used.

Reply:
I guess this DataConfig class is specific to Train/Tune. are there other cases that use this DataConfig, but not Train/Tune?